scheduler/test/cook/test/config.clj

```diff
@@ -123,6 +123,40 @@
                                                              :valid-models #{"valid-gpu-model"}
                                                              :default-model "invalid-gpu-model"}])))))
 
+(deftest test-disk-config-settings
+  (testing "empty disk"
+    (is (nil? (config/guard-invalid-disk-config []))))
+  (testing "valid default type"
+    (is (nil? (config/guard-invalid-disk-config [{:pool-regex "test-pool"
```

Comment:
People imitate unit tests. Should use ^test-pool$ here and down below.

Reply:
done